### PR TITLE
CR-1267 | updating logging for monitoring

### PR DESCRIPTION
**Pull request by akritisinhaa**

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java [hunk 1]
```diff
@@ -55,15 +55,15 @@ public void create() {
   @Override
   public void storeObject(final String schema, final String key, final Object value)
       throws CTPException, DataStoreContentionException {
-    log.with(schema).with(key).debug("Saving object to Firestore");
+    log.with(schema).with(key).info("Saving object to Firestore");
```
Comment (rpcatling) [inline src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java:58]: probably best to have named items here 
```
log.with("schema", schema).with("key", key).info( ....);
```

File: src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java [hunk 2]
```diff
@@ -55,15 +55,15 @@ public void create() {
...
 
     // Store the object
     ApiFuture<WriteResult> result = firestore.collection(schema).document(key).set(value);
 
     // Wait for Firestore to complete
     try {
       result.get();
-      log.with(schema).with(key).debug("Firestore save completed");
+      log.with(schema).with(key).info("Firestore save completed");
```
Comment (rpcatling) [inline src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java:66]: probably best to have named items here 
```
log.with("schema", schema).with("key", key).info( ....);
```

**Review by rpcatling — APPROVED** [review]
Just minor updates.

File: src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java (hunk 1 above)
Comment (akritisinhaa) [reply src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java]: done

File: src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java (hunk 2 above)
Comment (akritisinhaa) [reply src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java]: done

File: src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java [hunk 3]
```diff
@@ -55,15 +55,15 @@ public void create() {
...
+    log.with("schema", schema).with("key", key).info("Saving object to Firestore");
 
     // Store the object
     ApiFuture<WriteResult> result = firestore.collection(schema).document(key).set(value);
 
     // Wait for Firestore to complete
     try {
       result.get();
-      log.with(schema).with(key).debug("Firestore save completed");
+      log.with("schema", schema).with("key", schema).info("Firestore save completed");
```
Comment (cookel2) [inline src/main/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStore.java:66]: I think this says schema where it should say key. I suggest this:
```suggestion
      log.with("schema", schema).with("key", key).info("Firestore save completed");
```

**Review by cookel2 — CHANGES_REQUESTED** [review]
please see comment - just one small change suggested

**Review by cookel2 — APPROVED** [review]
looks good and builds successfully in travis now

**Pull request merged**